app/Http/Controllers/PurchaseOrderController.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\PurchaseOrder;
+use Illuminate\Http\Request;
+use Illuminate\Validation\ValidationException;
+
+class PurchaseOrderController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        try {
+            $request->validate([
+                'supplier_id' => "required|integer|min:0",
+                'status' => "required|string|in:pending,received",
```

Comment:
Looks like you missed a place to use $statuses! 😂

Reply:
I should have caught that one lol! Updating it right now.